extensions/wikia/GlobalNavigation/styles/GlobalNavigationInverse.scss

```diff
@@ -56,8 +57,15 @@ $global-nav-inverse-background-hover: #656E78;
 				border-radius: 50%;
 			}
 
-			.sign-in-label {
+			.auth-label,
+			.auth-link {
 				color: $global-nav-inverse-text-color;
+				&.register,
+				&.sign-in {
```

Comment:
This would get resolved to really long selector. Can you think about simplifying it?